extensions/resend/package.json

```diff
@@ -41,6 +44,13 @@
       "subtitle": "Resend",
       "description": "View Logged Emails and Send A New Email",
       "mode": "view"
+    },
+    {
+      "name": "contacts",
+      "title": "Contacts",
+      "subtitle": "Resend",
+      "description": "View, create, update and delete contacts from your audiences.",
+      "mode": "view"
```

Comment:
Maybe it would make more sense if we rename this to `Audiences and Contacts`? Currently, you are dealing only with Contacts but the API also supports Audiences now e.g. [Add Audience](https://resend.com/docs/api-reference/audiences/create-audience) so eventually we'll want to support Audience endpoints as well so easier to do it right now rather than change later.

Reply:
It might be better if we added a separate Audiences command, rather than adding more functionality to this one? That command could allow you to create, update and delete audiences, which would be a lot to fit into a single command. The only purpose of audiences for this command is allowing you to group / filter contacts.